app/views/users/show.html.haml

```diff
@@ -0,0 +1,42 @@
+%section.user.container
+  .row
+    .col-sm-10.col-sm-offset-1
+      %article
+        %header
+          %img/
+          %h2
+            = @user.full_name
+            's video collections
+            %span{"data-id" => "queue_items.count"}
```

Comment:
Is this a typo? `id` seems like an odd `data` key to store the number of queue items in.